### DISCUSSION TITLE
2011/04/11/%E8%B1%86%E8%B1%86%E7%9C%8B%E4%B9%A6%E6%97%B6%E5%83%8F%E6%88%91%EF%BC%9B%E7%9C%8B%E6%88%91%E6%97%B6%E5%83%8F%E6%88%91%E8%80%81%E5%A9%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#325] [Announcements]
# 2011/04/11/%E8%B1%86%E8%B1%86%E7%9C%8B%E4%B9%A6%E6%97%B6%E5%83%8F%E6%88%91%EF%BC%9B%E7%9C%8B%E6%88%91%E6%97%B6%E5%83%8F%E6%88%91%E8%80%81%E5%A9%86

眼睛看书时很小；看我时很大

https://blog.qizhen.xyz/2011/04/11/%E8%B1%86%E8%B1%86%E7%9C%8B%E4%B9%A6%E6%97%B6%E5%83%8F%E6%88%91%EF%BC%9B%E7%9C%8B%E6%88%91%E6%97%B6%E5%83%8F%E6%88%91%E8%80%81%E5%A9%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

luoqi999
2011年04月14日 13:15 编辑
以后就反过来了 。

回复
lizzie
2011年04月13日 17:48 编辑
越来越帅了

回复
勺子
2011年04月12日 12:54 编辑
豆豆看书时像我；看我时像我老婆
這名更貼嘛

回复
ruanqizhen
2011年04月12日 13:17 编辑
有道理啊，改回来

回复
